{"issue_type": "request_error", "error_msg": "HTTPConnectionPool(host='phoenixmunicipalcourt.gov', port=80): Max retries exceeded with url: /robots.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f77453a6430>, 'Connection to phoenixmunicipalcourt.gov timed out. (connect timeout=15)'))"}